### PR TITLE
Caller now can now pass error as string (old behavior) or dictionary

### LINKED ISSUE #38: Update passing `string` as error with `object`
instead of using `error` we need to pass object `{message: "error mesage", code: 1}`
@hboon could u please make pr to this repo with changes, needs for https://github.com/AlphaWallet/alpha-wallet-ios/issues/6456
we actually need to use
`const response = {'id': id, jsonrpc: '2.0', result: value, error: error }` instead of
`const response = {'id': id, jsonrpc: '2.0', result: value, error: {message: error} } `

### PR DESCRIPTION
**Pull request by hboon**
Close #38

### CODE REVIEW

File: dist/AlphaWallet.js [hunk 1]
```diff
@@ -39974,7 +39974,7 @@ var AlphaWallet = {
     var callback = callbacks[id];
 
     if (callback.isRPC) {
-      var response = { 'id': id, jsonrpc: '2.0', result: value, error: { message: error } };
+      var response = { 'id': id, jsonrpc: '2.0', result: value, error: error };
```
Comment (hboon) [inline dist/AlphaWallet.js:39977]: @oa-s is it better to check `error` if it's just a string and keep the old behavior?

```
if error is string //I forgot my JavaScript
    error: { message: error }
else
    error: error
end
```

Comment (oa-s) [reply dist/AlphaWallet.js:39977]: yes, looks good

**Review by oa-s — APPROVED** [review]

**Comment by hboon** [comment]
Force-pushed so the error passed to the callback can be a string (old behavior) as well as a dictionary

**Comment by hboon** [comment]
@oa-s Created https://github.com/AlphaWallet/alpha-wallet-ios/issues/6467

**Pull request merged**